app/assets/stylesheets/components/materials_bin.scss

```diff
@@ -69,4 +69,64 @@
     padding: 5px;
     border: 1px solid #ccc;
   }
+
+  .mb-toggle-info {
+    background: url('/assets/components/materials_bin/itsi_info.png') no-repeat;
```

Comment:
You should use the `image-url` sass helper for these, since it plays nicely with the rails assets pipeline:
`background: image-url('components/materials_bin/itsi_info.png') no-repeat;`